### ISSUE TITLE
Add support for python 3

### ISSUE THREAD

**Issue by emanueldima** [EUDAT-B2SAFE/B2HANDLE#92]
We exclusively use Python 3 in B2SHARE v2. Please add support for python 3 in the library. The first issue I have is the implicit relative import in package. So in Python 3 I get this error:

```
>>> from b2handle.handleclient import EUDATHandleClient
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/edima/.virtualenvs/b2share-evolution/lib/python3.5/site-packages/b2handle/handleclient.py", line 16, in <module>
    import utilhandle
ImportError: No module named 'utilhandle'
```

If unclear, see also this link: http://stackoverflow.com/questions/12172791/changes-in-import-statement-python3

**Comment by TobiasWeigel** [comment]
We have to see how Python 3 support fits into the roadmap. A quick test with 2to3 tells me that there is quite a bit of significant refactoring required to come to a source code base that is compatible with both py2 and py3.

**Comment by TobiasWeigel** [comment]
Short update. I've added an item on the WP5 development roadmap to look closer into the options we have for Python 3 support/migration after September.

**Comment by SofianeB** [comment]
This has been addressed in #116
